.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "files.exclude": {
```

Comment:
This will cause these folders to be NOT visible in VS Code UI...these were distracting